certified-connectors/Office365GroupsMail/apiDefinition.swagger.json

```diff
@@ -0,0 +1,768 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "description": "The Microsoft 365 Groups Mail connector allows users to work with data located in a Microsoft 365 Group mailbox. The functionality in this connector works independently from a single user's personal mailbox and allows users to work with any Group they can access within their organization. The connector works off the Microsoft Graph API, and uses terminology specified in a Microsoft 365 Groups documentation within the Graph API. Learn more in the connector documentation.",
+        "title": "Microsoft 365 Groups Mail",
+        "version": "1.0",
+        "x-ms-api-annotation": {
+            "status": "Preview"
+        },
+        "contact": {
+            "name": "Microsoft"
+        }
+    },
+    "host": "graph.microsoft.com",
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/v1.0/groups/{groupId}/conversations": {
+            "get": {
+                "operationId": "ListConversations",
+                "description": "Get all the conversations in this group.",
+                "summary": "List the conversations of a group",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/groupId__in_path"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Success",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "value": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/Conversation"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "Invalid group",
+                        "schema": {
+                            "type": "object"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
```

Comment:
How are you going to make paging work without policy or a policy template?

Reply:
My understanding is that `x-ms-pageable` does not needs neither of those. AzureAD uses it [[here](https://github.com/microsoft/PowerPlatformConnectors/blob/47eb4d2b5552594233b9e9feab366ff494a14ad7/certified-connectors/AzureAD/apiDefinition.swagger.json#L313)] and Office 365 Groups [[here](https://github.com/microsoft/PowerPlatformConnectors/blob/47eb4d2b5552594233b9e9feab366ff494a14ad7/custom-connectors/Office365Groups/apiDefinition.swagger.json#L327)]; am I missing something?

It is also defined in the apiDefinition.swagger schema [[here](https://github.com/microsoft/PowerPlatformConnectors/blob/47eb4d2b5552594233b9e9feab366ff494a14ad7/schemas/apiDefinition.swagger.schema.json#L346)].

Reply:
AzureAD uses policy template updatenextlink:
https://github.com/microsoft/PowerPlatformConnectors/blob/47eb4d2b5552594233b9e9feab366ff494a14ad7/certified-connectors/AzureAD/apiProperties.json

Reply:
If we have the next link property name, we can just add the same in this case.

Reply:
True, I have added the policy and tested the pagination. Thanks for catching this!